tests/functional/adapter/test_simple_seed.py

```diff
@@ -0,0 +1,123 @@
+import pytest
+from dbt.tests.adapter.simple_seed.fixtures import macros__schema_test
+from dbt.tests.adapter.simple_seed.seeds import (
+    seeds__enabled_in_config_csv,
+    seeds__tricky_csv
+)
+from dbt.tests.adapter.utils.base_utils import run_dbt
+
+_SCHEMA_YML = """
+version: 2
+seeds:
+- name: seed_enabled
+  columns:
+  - name: birthday
+    tests:
+    - column_type:
+        type: date
+  - name: seed_id
+    tests:
+    - column_type:
+        type: character varying(256)
+
+- name: seed_tricky
+  columns:
+  - name: seed_id
+    tests:
+    - column_type:
+        type: integer
+  - name: seed_id_str
+    tests:
+    - column_type:
+        type: character varying(256)
+  - name: a_bool
+    tests:
+    - column_type:
+        type: boolean
+  - name: looks_like_a_bool
+    tests:
+    - column_type:
+        type: character varying(256)
+  - name: a_date
+    tests:
+    - column_type:
+        type: timestamp without time zone
+  - name: looks_like_a_date
+    tests:
+    - column_type:
+        type: character varying(256)
+  - name: relative
+    tests:
+    - column_type:
+        type: character varying(9)
+  - name: weekday
+    tests:
+    - column_type:
+        type: character varying(8)
+""".lstrip()
+
+
+class TestSimpleSeedColumnOverride:
```

Comment:
Since redshift is so similar to postgres can we just inherit `TestSimpleSeedColumnOverride` from dbt-core?

Reply:
I did that originally but each adapter expects different types: [Failed run](https://github.com/dbt-labs/dbt-redshift/actions/runs/4089361462/jobs/7051976514)

Reply:
could you inherit and override the models macro to include the right types?

Reply:
I'll try that out.